.github/workflows/ci.yml

```diff
@@ -19,62 +19,118 @@ jobs:
         include:
```

Comment:
open comment, not obligatory for this PR

as we are here, we can also consider stages approach, eg
https://github.com/php-coveralls/php-coveralls/actions/runs/5566754160
<img width="965" alt="Screenshot 2023-10-25 at 00 46 58" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/2716794/a3ad5350-ba4f-4ead-9359-af9964b7aa61">

Reply:
Yeah, I was also considering this, but I don't have experience with GitHub's stages approach and need to dig it before. I am not sure how properly utilise this for job matrix, maybe we run job for each supported PHP version to prepare machines and then reuse them in the actual jobs for running tests/Fixer? Anyway, I don't want to do it in this PR 🙂.